### PR TITLE
Add Session Loading style to list of known configs

### PR DESCRIPTION
**Pull request by fendor**
See https://github.com/haskell/haskell-language-server/issues/4178 for details.

### CODE REVIEW

**Comment by soulomoon** [comment]
🤔 why mainly break on windows

**Comment by fendor** [comment]
It claims the extension https://marketplace.visualstudio.com/items?itemName=justusadam.language-haskell is unknown on windows.

I didn't get to it yet, I suppose it is simply a CI error.

File: package.json [hunk 1]
```diff
@@ -210,6 +210,20 @@
...
         },
+        "haskell.sessionLoading": {
+          "scope": "resource",
+          "type": "string",
+          "enum": [
+            "singleComponent",
+            "multipleComponents"
+          ],
+          "default": "singleComponent",
+          "description": "Project loading preference of the Haskell Language Server. It allows the build tool (such as `cabal` or `stack`) to [load multiple components at once](https://github.com/haskell/cabal/pull/8726). While this can lead to an improved IDE experience, it is an experimental feature.",
```
Comment (michaelpj) [inline package.json:221]:
```suggestion
          "description": "Preferred approach for loading package components. Setting this to 'multiple components' (EXPERIMENTAL) allows the build tool (such as `cabal` or `stack`) to [load multiple components at once](https://github.com/haskell/cabal/pull/8726), which is a significant improvement.",
```

**Review by michaelpj — APPROVED** [review]
how about this?

File: package.json (hunk 1 above)
Comment (fendor) [reply package.json]: Thanks!

**Pull request merged**